backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap.cuh

```diff
@@ -113,7 +113,7 @@ mul_ggsw_glwe(Torus *accumulator, double2 *fft, double2 *join_buffer,
 
```

Comment:
In this file, this section:
```
  // accumulate rest of the products into fft buffer
  for (int l = 1; l < gridDim.x; l++) {
    auto cur_src_acc = get_join_buffer_element<G>(l, blockIdx.y, group,
                                                  join_buffer, polynomial_size,
                                                  glwe_dimension, support_dsm);
    tid = threadIdx.x;
    for (int i = 0; i < params::opt / 2; i++) {
      fft[tid] += cur_src_acc[tid];
      tid += params::degree / params::opt;
```
could be done on blocks whose blockIDx.x = 0 only as well no?